design docs/durandal play by play.txt

```diff
@@ -0,0 +1,55 @@
+
+After careful consideration, Player1 signs the contract. They experience reduced production overhead but retain significant overhead costs due to their intertwined internal structures.
+
+With commercial satellites bundled, launches are freed for a new project. Though a moon mining project is desirable, the budget remains insufficient. Player1 opts for an intermediate project: constructing additional modules for the ISS-2 to rent to corporations for increased revenue.
+
+Player1 instructs available engineers to design modules optimized for tourism. Leveraging their specialization, the design is efficiently completed. Player1 accumulates some debt to launch these modules. Corporations submit bids with various requirements, influencing long-term relationships and reputation. Player1 accepts a bid offering substantial payment and operational integration with Player1's existing infrastructure, effectively creating a profitable space hotel.
+
+Due to high visibility and successful operation management, Player1's popularity significantly rises, directly increasing government funding. This enhanced reputation also attracts better future partnership bids and improved loan terms.
+
+With improved funding, the moon mining operation becomes feasible. However, significant initial funding gaps and no available launches require extensive restructuring. Due to the agency's specialization as an Old Space Agency, the restructuring process incurs additional costs and complexity because of internal resistance and inflexibility. Player1 privatizes satellite launch operations, clearly losing substantial experience in rocketry, requiring considerable time, investment, and effort to recover lost capabilities and integrate new operational frameworks.
+
+Player1 initiates detailed moon mining project planning. They list clearly customizable project components:
+
+A. Mining Infrastructure
+B. Thermal Extraction Systems
+C. Helium-3 Processing & Separation
+D. Power Generation & Storage (options impact reliability and maintenance costs)
+E. Launch & Transport Systems
+F. Habitat & Life Support
+G. Construction & Assembly Systems
+H. Control, Autonomy & Robotics
+I. Communications & Navigation
+J. Laboratory & Monitoring Equipment
+K. Logistics & Storage
+L. Safety, Redundancy & Emergency Systems
+M. Earth-Based Support Infrastructure
+
+Player1 can directly fulfill departments E, F, G, and M. The rest are opened for bidding, clearly influencing long-term project reliability, operational costs, and potential crises. Player1 defines interoperability standards and strategic choices:
+
+Minimum life-support standards for all departments, enhancing crew maintenance ease but discouraging purely autonomous bids.
+
+Centralized power generation system, clearly communicating single-point-of-failure risks and cost-saving trade-offs.
```

Comment:
In-game effect of the latter half being? This sounds like a management memo rather than a 4x game action.